src/PrestaShopBundle/Form/Admin/Product/ProductCombination.php

```diff
@@ -124,29 +124,42 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ))
         ->add('attribute_weight', 'Symfony\Component\Form\Extension\Core\Type\NumberType', array(
             'required' => false,
-            'label' => $this->translator->trans('Impact on weight', [], 'Admin.Catalog.Feature')
+            'label' => $this->translator->trans('Impact on weight', array(), 'Admin.Catalog.Feature')
         ))
         ->add('attribute_unity', 'Symfony\Component\Form\Extension\Core\Type\MoneyType', array(
             'required' => false,
-            'label' => $this->translator->trans('Impact on price per unit (tax excl.)', [], 'Admin.Catalog.Feature'),
+            'label' => $this->translator->trans('Impact on price per unit (tax excl.)', array(), 'Admin.Catalog.Feature'),
             'currency' => $this->currency->iso_code,
             'attr' => ['class' => 'attribute_unity'],
         ))
         ->add('attribute_minimal_quantity', 'Symfony\Component\Form\Extension\Core\Type\NumberType', array(
             'required' => false,
-            'label' => $this->translator->trans('Min. quantity for sale', [], 'Admin.Catalog.Feature'),
+            'label' => $this->translator->trans('Min. quantity for sale', array(), 'Admin.Catalog.Feature'),
             'constraints' => array(
                 new Assert\NotBlank(),
                 new Assert\Type(array('type' => 'numeric')),
             )
         ))
+        ->add('attribute_low_stock_threshold', 'Symfony\Component\Form\Extension\Core\Type\NumberType', array(
+            'label' => $this->translator->trans('Low stock level', array(), 'Admin.Catalog.Feature'),
```

Comment:
Confirming we need to have Admin.Catalog.Feature as a domain for "Low stock level" :)